.github/workflows/ci.yaml

```diff
@@ -350,13 +358,17 @@ jobs:
            github.event_name == 'pull_request'
            && !contains(github.event.pull_request.body, '[skip ci]')
            && !contains(github.event.pull_request.body, '[skip community_build]')
+           && !contains(github.event.pull_request.body, '[skip community_build_c]')
          )
          || (
            github.event_name == 'workflow_dispatch'
            && github.repository == 'lampepfl/dotty'
          )"
 
     steps:
+      - name: Set JDK 17 as default
+        run: echo "/usr/lib/jvm/java-17-openjdk-amd64/bin" >> $GITHUB_PATH
```

Comment:
I don't think Java 17 is installed on the docker image we're currently running, so this might not do anything

Reply:
The Dockerfile was updated in October 2021: https://github.com/lampepfl/dotty-ci/pull/19, but the image we're running is from March 2021: https://github.com/lampepfl/dotty/blob/main/.github/workflows/ci.yaml#L37

Reply:
I changed it to JDK 16. We can update it when we update the image.

How do we update that image?

Reply:
To push a new image, see instructions in https://github.com/lampepfl/dotty-ci#dotty-docker-image, then the ci.yaml file can be updated to point to the new one.